src/ReactiveUI/ReactiveBinding.cs

```diff
@@ -150,9 +141,6 @@ internal class ReactiveBinding<TView, TViewModel, TValue> : IReactiveBinding<TVi
         /// </value>
         public BindingDirection Direction { get; private set; }
 
-        /// <summary>
-        /// Releases unmanaged and - optionally - managed resources.
-        /// </summary>
         public void Dispose()
```

Comment:
I don't really know...it seemed a bit like stating the obvious. To be honest, I'm not sure any of these comments make sense on the `internal` implementation of the fairly well commented interface...